src/update.rs

```diff
@@ -136,3 +147,39 @@ impl DummyOpenOptionsExt for OpenOptions {
         self
     }
 }
+
+/// A naive way of checking if fp is installed through homebrew.
+///
+/// This will check if the current executable is located in the default linux
+/// homebrew location: `/home/linuxbrew/.linuxbrew`. This will give a false
+/// negative if the if the user has changed the homebrew path.
+#[cfg(target_os = "linux")]
+#[inline(always)]
```

Comment:
do we really wanna force inline this function? we can just do
```suggestion
#[inline]
```
and the compiler will use smartness to determine if it makes sense